cpuinfo_test.go

```diff
@@ -402,3 +465,16 @@ func TestCPUInfoParseRISCV64(t *testing.T) {
 		t.Errorf("want ModelName %v, have %v", want, have)
 	}
 }
+
+func TestCPUInfoParseLOONG64(t *testing.T) {
+	cpuinfo, err := parseCPUInfoloong64([]byte(cpuinfoloong64))
+	if err != nil || cpuinfo == nil {
+		t.Fatalf("unable to parse loongarch cpu info: %v", err)
+	}
+	if want, have := 4, len(cpuinfo); want != have {
+		t.Errorf("want number of processors %v, have %v", want, have)
+	}
+	if want, have := "loongarch32 loongarch64", cpuinfo[1].ModelName; want != have {
+		t.Errorf("want ModelName '%v', have '%v'", want, have)
+	}
+}
```

Comment:
Please adjust this accordingly.

Also obviously the "ModelName" should not be "loongarch32 loongarch64", this part need fixing in `parseCPUInfoLoong` as well.